src/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+
+  &__star {
+    display: flex;
+    margin-right: 4px;
+
+    width: $star-size;
+    height: $star-size;
+
+    background: url(../images/star.svg) center no-repeat;
+  }
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  width: $star-size;
+  height: $star-size;
+
+  background: url(../images/star-active.svg) center no-repeat;
+}
```

Comment:
- Use nesting
- width and height you have already set to the `&__star` selector and they are the same